src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=<enter operator name here>
+publisher=<enter publisher name here>
+wat.warcinfo.description=<enter warc info description here>
```

Comment:
These defaults will put strings like "<enter operator name here>" in the WAT file if the user doesn't think to edit the commons. properties file (is there documentation anywhere on using the WAT extractor?). One option to help this could be to leave the values for operator, publisher, and wat.warcinfo.description empty, and if at WAT writing time they are still empty, don't include the fields in the warcinfo content since the fields are optional anyway.